diff --git a/dist/development/chunk-JMJ3UQ3L.mjs b/dist/development/chunk-JMJ3UQ3L.mjs
index 67db8da6fba2312d84f545bf0dd0acc8c3387bdc..a61e10fbdabab3e85c00024836f17798e672fa63 100644
--- a/dist/development/chunk-JMJ3UQ3L.mjs
+++ b/dist/development/chunk-JMJ3UQ3L.mjs
@@ -9494,6 +9494,7 @@ import(${JSON.stringify(manifest.entry.module)});`;
     {
       "rr-importmap": "",
       type: "importmap",
+      nonce: scriptProps.nonce,
       suppressHydrationWarning: true,
       dangerouslySetInnerHTML: {
         __html: JSON.stringify({
